### PR TITLE
fix(unittests): make unittests pass on Windows per issue #538

### PR DESCRIPTION
**Pull request by nicolatl**
### Description
Unittests now pass on windows. 

### Changes
- Use path.join on all path bindings
- Use cross-env to set env vars

### Checks
- [ ] Project builds
- [ ] Project passes unittests
- [ ] Updated documentation accordingly

### Notes
@shawnzhang12 and I got 36/39 integration tests passing too, but didn't include those in this PR, to keep PRs modular. This integration work is in a fork here https://github.com/shawnzhang12/booster/commits/main.

### CODE REVIEW

**Review by NickSeagull — CHANGES_REQUESTED** [review]
Awesome! Just as a nit-pick, why `path.join` and not `path.normalize`?

**Review by javiertoledo — APPROVED** [review]
Great work @nicolatl!

**Comment by nicolatl** [comment]
> Awesome! Just as a nit-pick, why `path.join` and not `path.normalize`?

Good point -- normalize makes more sense. I changed this where applicable a960e8cc050874f0db53265e01e122f40b9d3e9e

File: packages/framework-provider-aws-infrastructure/package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "clean": "rimraf ./dist tsconfig.tsbuildinfo",
     "prepack": "tsc -b tsconfig.json",
     "test:provider-aws-infrastructure": "npm run test",
-    "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
+    "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\" --timeout 600000"
```
[the remark this answers is not in the document]
Comment (nicolatl) [reply packages/framework-provider-aws-infrastructure/package.json]: Great catch -- this was my temporary solution for aws-infrastructure timeouts, which I was seeing as per issue https://github.com/boostercloud/booster/pull/1267.
A better solution could be found for that issue, or in any case it should be its own PR.

**Review by NickSeagull — APPROVED** [review]

**Pull request merged**